modules/seqexec/server/src/main/resources/Tcs.xml

```diff
@@ -209,11 +209,6 @@
         </command>
         <command name="odgw3Park">
             <description>ODGW 3 park</description>
-            <parameter name="command">
-                <channel>odgw3Park.A</channel>
-                <type>STRING</type>
-                <description>Park command</description>
-            </parameter>
```

Comment:
This parameter comes from the old Seqexec, where they were used to mark the CAD record. The new Seqexec can mark the record using the DIR input, so this parameter is no longer necessary.